src/qae/object-properties.js

```diff
@@ -26,6 +26,8 @@ const properties = {
     qMeasures: [],
     /** @type {number[]} */
     qColumnOrder: [],
+    /** @type {number[]} */
```

Comment:
I think the jsdoc tag should be removed as we still don't want it to be a part of the API spec.

Also I wonder if a better fix is to actually remove the usage of it from the "ext/data" code instead.

Reply:
why do we need it for the chart conversion? I think we need to map `columnWidths` to the corresponding `columnWidth` on each column's `qDef`, and vice versa, when converting between old and new table

Reply:
In the main branch, there is no `columnWidths: []`, after we change the columnWidths to the old table, we cannot convert it to new table.
Same for the new table, after we change the columnWidth to the new table, we cannot convert it to the old table. 
![Screenshot 2023-03-31 at 11 37 49](https://user-images.githubusercontent.com/4471489/229449002-611f789b-0ef4-4f2f-bdd3-741971512243.png)
![Screenshot 2023-03-31 at 11 37 18](https://user-images.githubusercontent.com/4471489/229449045-9aebfd3e-c847-484e-be7d-e3468838fc9e.png)

Reply:
Yes, it is